api/src/components/reference/service.ts

```diff
@@ -17,18 +18,43 @@ export const getAllByPublication = async (publicationId: string) => {
         }
     });
 
-    return references;
+    // Return with publicationId.
+    return references.map((reference) => {
+        const { publicationVersionId, ...referenceRest } = reference;
+
+        return { publicationId, ...referenceRest };
```

Comment:
Is the "publicationId" really needed? The return type of this service doesn't match any model into our DB if we combine it like that. We will end up with scenarios where we need to get rid of this "publicationId"